### PR TITLE
feat:  Profile Setup(#40012)

### PR DESCRIPTION
**Pull request by ajayrose25**

### CODE REVIEW

File: src/ controllers/user/routes.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+.delete(authMiddleware(permissions.getUsers, 'all'), UserController.delete)
+.put(authMiddleware(permissions.getUsers, 'all'), validationHandler(validation.update), UserController.Update);
+userRouter.delete('/:id', validationHandler(validation.delete), UserController.delete);
+userRouter.get('/:id', authMiddleware(permissions.getUsers, 'all'), validationHandler(validation.get), UserController.getAll);
+
+//userRouter.route('/new/me')
+//.get(authMiddleware(permissions.getUsers, 'all'),(req: IRequest,res)=>{
+  //  console.log('inside routes>>>>>>>>>>>>>>>>>>>>>');
+    //res.send(req.user);
+//})
```
Comment (VisaaliSuccessive) [inline src/ controllers/user/routes.ts]: uncomment the useful code

File: src/libs/routes/authMiddleware.ts [hunk 2]
```diff
@@ -18,31 +20,46 @@ export default (module, permissionType) => (req: Request, res: Response, next: N
...
-        message: 'Permission Denied'
+    const { _id, email } = decodedUser;
+    await userRepository
+      .getById({_id,email})
+      .then(user => {
+        if (!user) {
+          next({
+            status: 403,
+            error: 'Unauthorized Access',
+            message: 'User does not Exist in the System'
```
Comment (VisaaliSuccessive) [inline src/libs/routes/authMiddleware.ts]: Message must be Permission Denied

File: src/ controllers/user/routes.ts (hunk 1 above)
Comment (ajayrose25) [reply src/ controllers/user/routes.ts]: Done

File: src/libs/routes/authMiddleware.ts (hunk 2 above)
Comment (ajayrose25) [reply src/libs/routes/authMiddleware.ts]: Done

**Review by VisaaliSuccessive — APPROVED** [review]

**Pull request merged**